strongbox-web-core/src/main/java/org/carlspring/strongbox/controller/UserController.java

```diff
@@ -211,7 +203,7 @@ ResponseEntity update(@RequestParam(value = "juser",
     ResponseEntity delete(@ApiParam(value = "The name of the user") @PathVariable String name)
             throws Exception
     {
-        User user = userService.findByUserName(name);
+        User user = databaseTx.detach(userService.findByUserName(name), true);
         if (user == null || user.getId() == null)
         {
             return toError("The specified user does not exist!");
```

Comment:
Just use detachAll() instead of detach() all the time.

http://orientdb.com/docs/2.1/Object-Database.html#detach
http://orientdb.com/docs/2.1/Object-Database.html#lazy-detachall